app/commands.py

```diff
@@ -119,11 +119,13 @@ def create_api_key(client_id):
 
 
 @app.cli.command("run_certificate", with_appcontext=True)
+@click.argument("nb_fork", type=click.INT)
 @click.argument("as_of_date", required=False)
-def run_certificate(as_of_date=None):
+def run_certificate(nb_fork, as_of_date=None):
```

Comment:
:orange_circle: you could use multiprocessing.cpu_count() to know how many cpu you have on the current machine, so you don't have to pass nb_fork as parameter.